src/repr/scalar/datetime.rs

```diff
@@ -2223,74 +2499,54 @@ mod test {
             (
                 "1 year 2 years",
                 Second,
-                "Invalid INTERVAL '1 year 2 years': YEAR field \
-                set twice",
+                "YEAR field set twice",
             ),
             (
                 "1-2 3-4",
                 Second,
-                "Invalid INTERVAL '1-2 3-4': YEAR or MONTH field \
-                set twice",
+                "YEAR or MONTH field set twice",
```

Comment:
this doesn't _need_ to be changed, but not including the original interval in the error message seems slightly worse when debugging parsing a collection from a relation.

Reply:
The original intervals gets added to the error message in `strconv::parse_interval_w_disambiguator()`; these tests are simply grabbing the errors from `ParsedDateTime::build_parsed_datetime_interval`. I was trying to make this error decorating consistent across all of the timelike types, many of which use the same underlying functions. lmk if this is still no good.